test/Regex/characterclass_with_range.js

```diff
@@ -0,0 +1,132 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+function testRegExp(str, regexp, expectedResult)

```

Comment:
for `expectedResult`, instead of a Boolean, you could test with `.match` and have an expected value for `match[0]`

Reply:
changed testRegExp to matchRegExp and it now verifies the matched string.